css/index.scss

```diff
@@ -63,46 +65,42 @@ button{
   padding: 13px 20px;
   border: none;
   border-radius: 5px;
-}
 
-button:hover{
-  background-color: #ccc;
-}
-
-input[type=text] {
-    padding: 12px 20px;
-    margin: 8px 0;
-    display: inline-block;
-    border: 1px solid #ccc;
-    border-radius: 5px;
-    box-sizing: border-box;
-  }
-  
-  input[type=submit] {
-    background-color: white;
-    color: black;
-    padding: 13px 20px;
-    margin: 8px 0;
-    border: none;
-    border-radius: 5px;
-    cursor: pointer;
+  &:hover{
+    background-color: #ccc;
   }
+}
   
-  input[type=submit]:hover {
+input[type=submit] {
+  background-color: white;
+  color: black;
+  padding: 13px 20px;
+  margin: 8px 0;
+  border: none;
+  border-radius: 5px;
+  cursor: pointer;
+
+  &:hover{
     background-color: #ccc;
-  }
+  }  
+}
 
-  
-  input[type=button] {
-    width: 100%;
-    padding: 12px 20px;
-    margin: 8px 0;
-    display: inline-block;
-    border: 1px solid #ccc;
-    border-radius: 5px;
-    box-sizing: border-box;
-  }
+input[type=text], input[type=button]{
+  padding: 12px 20px;
+  margin: 8px 0;
+  display: inline-block;
+  border: 1px solid #ccc;
+  border-radius: 5px;
+  box-sizing: border-box;
+}
+
+input[type=button] {
+  width: 100%;
+  border-radius: 5px;
+  box-sizing: border-box;
+}
```

Comment:
The cursor doesn't change to a pointer when I have my mouse over the button. Should add `cursor: pointer` like you did in `input[type:submit]`

Reply:
Umm, seems like the code was redundant....

Added thte cursor but nothing happend, changed it on the button class and it worked, started testing what this actully did, and it seems like nothing, dont know how...

But think I removed it all!